workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md

```diff
@@ -293,6 +350,8 @@ kubectl patch deployment tiller-deploy \
     }]'
 ```
 
+**NOTE:** I would recommend removing the line breaks and whitespace from this command.
+
 :question: 🤔
```

Comment:
The thinking face looks a little like a to-do. The question mark one from above is good - could you add a similar comment here?